ScreenConfig.proto

```diff
@@ -0,0 +1,48 @@
+syntax = "proto3";
+
+import "brewblox.proto";
+import "nanopb.proto";
+
+package blox.ScreenConfig;
+
+enum Type {
```

Comment:
Overly generic name. `NodeType` maybe?

Reply:
I've decided on moving this into layoutnode so namespacing will do the work.

Changed in f18dc1a6cdfbbd4f489edf5f8b2de641368e5911